lms/djangoapps/commerce/api/v1/views.py

```diff
@@ -19,12 +24,13 @@ class CourseListView(ListAPIView):
     authentication_classes = (OAuth2Authentication, SessionAuthentication,)
     permission_classes = (IsAuthenticated,)
     serializer_class = CourseSerializer
+    pagination_class = None
 
     def get_queryset(self):
-        return Course.iterator()
+        return list(Course.iterator())
```

Comment:
Removing this causes test failures.  I forget the exact details, but something in DRF v3.1 is calling a method that's defined on a genuine `Queryset` (and a `list`) but not on an iterator.